client/my-sites/comments/comment-navigation/index.jsx

```diff
@@ -315,7 +315,7 @@ export class CommentNavigation extends Component {
 
 					{ hasComments && (
 						<Button compact onClick={ toggleBulkMode }>
-							{ translate( 'Bulk Edit' ) }
+							{ translate( 'Bulk edit' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **1** times:
`translate( 'Bulk Edit', 'NULL', { context: 'NULL'} )` *ES Score: 10*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/button-sentence-case)